sale_order_warn_message/models/sale_order.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    sale_warn_msg = fields.Text(compute="_compute_sale_warn_msg")
+
+    @api.depends(
+        "state", "partner_id.sale_warn", "partner_id.commercial_partner_id.sale_warn"
+    )
+    def _compute_sale_warn_msg(self):
+        for rec in self:
+            if rec.state not in ["draft", "sent"]:
+                rec.sale_warn_msg = ""
+                continue
+            p = rec.partner_id.commercial_partner_id
+            sale_warn_msg = ""
+            separator = ""
+            if p.sale_warn == "warning":
+                separator = "\n"
+                sale_warn_msg += p.sale_warn_msg
+            if p != rec.partner_id and rec.partner_id.sale_warn == "warning":
+                sale_warn_msg += separator + rec.partner_id.sale_warn_msg
+            rec.sale_warn_msg = sale_warn_msg
```

Comment:
would it be better like this?
<img width="571" alt="Captura de pantalla 2020-05-12 a las 15 47 27" src="https://user-images.githubusercontent.com/22913794/81699631-e8699f80-9467-11ea-93af-11a75250b19f.png">

Reply:
It's exactly the same. Your code only has the problem that its more indented than mine because you simply are trying to avoid that "continue". Have any problem the use of the "continue"?

Reply:
IMO It's a good practice use only 1 write. I don't like the "continue" haha but if it's okay for all I don't hace problem with it, I like to let a loop finish naturally.

Reply:
Hi @MiquelRForgeFlow are you planning to work on these comments?

> IMO It's a good practice use only 1 write. I don't like the "continue" haha but if it's okay for all I don't hace problem with it, I like to let a loop finish naturally.

Reply:
> Hi @MiquelRForgeFlow are you planning to work on these comments?

No, because it's an optional comment regarding aesthetic indentation and I prefer the way it's already. If in the future, some migrator change that indentation with another equivalent, well, I won't oppose.